crates/node/src/catchup.rs

```diff
@@ -196,6 +213,8 @@ impl Node {
                     application_id: None,
                     last_executed_transaction_hash: calimero_primitives::hash::Hash::default(),
                     batch_size: self.network_client.catchup_config.batch_size,
+                    path: None,
```

Comment:
And this

Reply:
same here, we already have `application_id`